apps/entity-service/src/entities/processors/financial-report.processor.ts

```diff
@@ -23,6 +29,57 @@ export class FinancialReportProcessor extends ReportProcessor<
   readonly LIGHT_DTO = FinancialReportLightDto;
   readonly FULL_DTO = FinancialReportFullDto;
 
+  /**
+   * Specific method for FinancialReport custom logic. This is called automatically when nothingToReport is updated
+   */
+  protected async processFinancialReportSpecificLogic(model: FinancialReport): Promise<void> {
+    const organisation = await Organisation.findByPk(model.organisationId);
+    if (organisation == null) {
+      console.warn(`Organisation not found for FinancialReport ${model.uuid}`);
```

Comment:
Please don't use `console`. Look for other classes that create an instance of TMLogger for an example.